### ISSUE TITLE
gsub NoMethodError on Circle Post check

### ISSUE THREAD

**Issue by KGmajor** [orbit-love/community-ruby-circle-orbit#2]
Hey there! Thanks for putting this together, I eagerly ran this as a github action and the comments portion flowed flawlessly. 

Hit a snag on the Post check run. Looks like we need a Nil check before sanitizing the body with gsub. I'd open a PR, but I'm not a rubyist 😭 

Traceback:
```ruby 
Run circle_orbit --check-posts
  circle_orbit --check-posts
  shell: /usr/bin/bash -e {0}
  env:
    CIRCLE_API_KEY: ***
    CIRCLE_URL: ***
    ORBIT_API_KEY: ***
    ORBIT_WORKSPACE_ID: ***
/opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/interactions/post.rb:71:in `sanitize_body': undefined method `gsub' for nil:NilClass (NoMethodError)
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/interactions/post.rb:11:in `initialize'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/orbit.rb:10:in `new'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/orbit.rb:10:in `call'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/circle.rb:20:in `block (2 levels) in process_posts'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/circle.rb:19:in `each'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/circle.rb:19:in `block in process_posts'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/circle.rb:16:in `each'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/circle.rb:16:in `process_posts'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/lib/circle_orbit/client.rb:49:in `posts'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/scripts/check_posts.rb:13:in `render'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/base.rb:485:in `start'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/circle_orbit-0.0.1/bin/circle_orbit:37:in `<top (required)>'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/bin/circle_orbit:23:in `load'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/bin/circle_orbit:23:in `<main>'
Checking for new Circle Spaces posts and sending them to your Orbit workspace...
Error: Process completed with exit code 1.```

**Comment by bencgreenberg** [comment]
Hi @KGmajor!

Thanks so much for raising this. You are right, that's exactly what this needs. I'll put this together and publish a v0.0.2 release today.

Ben

**Comment by bencgreenberg** [comment]
I believe this has now been resolved in #3. I would wait about 20 minutes or so just to be on the safe side for the new gem version to make its way through Rubygems before running the Action again.
